test/postgres/setup-postgres.sh

```diff
@@ -18,6 +18,8 @@ ssl_key_file = '/share/secrets/postgres.key'
 ssl_ca_file = '/share/secrets/ca.crt'
 hba_file = '/share/conf/pg_hba.conf'
 wal_level = logical
+fsync = off
+synchronous_commit = off
```

Comment:
There are failiures already , so you may wish to try with just `fsync` in case the `synchronous_commit` is CI-unfriendly. I hope the speedup observed remains with just fsync off.